client/public/elements/pages/person/rp-page-person.tpl.js

```diff
@@ -385,13 +387,15 @@ return html`
         `}
 
       </section>
+    ${this.totalPublications != 0 || (this.isLoggedIn && this.isOwnProfile) ? html`
+        <ae-publication-list
```

Comment:
@jrmerz  This is the question I was asking about whether hiddenness is a function of the component or outside of it.  This implies it's not part of the component.

Reply:
I'm gonna just pull this in, and if we move that into the component completely, it's a pretty small fix.